Gemfile

```diff
@@ -66,3 +66,5 @@ gem 'devise'
 gem 'rubocop', require: false
 
 gem 'simple_form'
+
+gem "factory_girl_rails", "~> 4.0"
```

Comment:
This could go into a group with `:test, :development`.

Groups in a gemfile determine which environments a gem is loaded in. We won't need factorygirl in production.

See: http://yehudakatz.com/2010/05/09/the-how-and-why-of-bundler-groups/

:smile